install/playbooks/roles/certificates/files/renewal-hook.sh

```diff
@@ -16,10 +16,12 @@ for fqdn in $RENEWED_DOMAINS; do
         restart_nginx=$(grep -Rl "server_name $fqdn" /etc/nginx/sites-enabled/ | grep -v -e -cert -c)
     fi
 
+    # Make sure the full chain is readable by all programs
+    chmod a+r /etc/letsencrypt/archive/$fqdn/fullchain*.pem
```

Comment:
I am not sure yet why I needed to do this to have LDAP authentication working with Dovecot.
It seems dovecot and postfix need to read the fullchain in the ldap folder.